src/utils/front/hamburgerMenu.ts

```diff
@@ -0,0 +1,52 @@
+import {
```

Comment:
should we put it close to the HamburgerMenu code? it's not generic enough unless I'm missing something to keep it outside

Reply:
I can do that! I just placed it next to the other hooks by default. It probably makes more sense to keep all three in the same module since they must be used together to work properly.